src/overlays/actors/ovl_En_Bird/z_en_bird.c

```diff
@@ -47,81 +47,78 @@ void EnBird_Init(Actor* thisx, PlayState* play) {
     SkelAnime_Init(play, &this->skelAnime, &gBirdSkel, &gBirdFlyAnim, NULL, NULL, 0);
     ActorShape_Init(&this->actor.shape, 5500, ActorShadow_DrawCircle, 4);
     this->unk_194 = 0;
-    this->unk_198 = 0;
-    this->unk_1C0 = 0x9C4;
+    this->timer = 0;
+    this->rotYStep = 2500;
     this->actor.colChkInfo.mass = 0;
-    this->unk_1A8 = 1.5f;
-    this->unk_1AC = 0.5f;
-    this->unk_1A0 = 0.0f;
-    this->unk_1A4 = 0.0f;
-    this->unk_1B8 = 0.0f;
-    this->unk_1B0 = 40.0f;
+    this->speedXZTarget = 1.5f;
+    this->speedXZStep = 0.5f;
+    this->posYMag = 0.0f;
+    this->rotYMag = 0.0f;
+    this->posYWaveDelta = 0.0f;
+    this->flightDistance = 40.0f;
     this->unk_1BC = 70.0f;
-    func_809C1CAC(this, this->actor.params);
+    EnBird_Reset(this, this->actor.params);
 }
 
 void EnBird_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_809C1CAC(EnBird* this, s16 params) {
+void EnBird_Reset(EnBird* this, s16 params) {
```

Comment:
This looks like a setup function for `EnBird_StopMove`

Note `EnBird_StopMove` is active for 5-35 frames but it only takes like 3 frames for it to reach speedXZ 0 afaict
Maybe its purpose is more the bird is idling/not moving than the slowing down itself

so I would rename `EnBird_StopMove` to something but idk. Maybe Idle. and `EnBird_Reset` -> SetupIdle :shrug: ?